.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: Scala tests with coverage
+
+on:
+  pull_request:
+    paths-ignore: ['**.md']
+  push:
+    paths-ignore: ['**.md']
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mysql/mysql-server:5.6
+        env:
+          MYSQL_ROOT_PASSWORD: rawls-test
+          MYSQL_USER: rawls-test
+          MYSQL_PASSWORD: rawls-test
+          MYSQL_DATABASE: testdb
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 3307:3306
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      # coursier cache action caches both coursier and sbt caches
+      - name: coursier-cache-action
+        uses: coursier/cache-action@v5
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Git secrets setup
+        run: |
+          git clone https://github.com/awslabs/git-secrets.git ~/git-secrets
+          cd ~/git-secrets
+          git checkout ad82d68ee924906a0401dfd48de5057731a9bc84
+          sudo make install
+
+      - name: Secrets check
+        run: |
+          ./minnie-kenny.sh --force
+          git secrets --scan-history
+
+      - name: Run tests
+        run: >-
+          sbt coverage test
+          -J-Xmx3g
+          -Denv.type=test
+          -Dmysql.host=localhost
+          -Dmysql.port=3307
+          -Ddirectory.url=ldap://localhost:3389
+          -Ddirectory.password=testtesttest
+          it:compile
+
+      - uses: codecov/codecov-action@v1
```

Comment:
something - which I haven't tried to debug - is not working with code coverage reports. I see "There was an error processing coverage reports." when viewing https://codecov.io/gh/broadinstitute/rawls/commit/05b05ece21a932b52ef9a7e1a1a83d43ec84e189. Could you look into this? (maybe possibly it's the removal of `coverageAggregate`?)

Reply:
@Qi77Qi and I looked at the reports - coverageAggregate wasn't actually generating any reports, because it required a report generation step from the underlying individual modules first, and that wasn't happening, so it was a no-op. I'm adding `coverageReport`, I'm not sure why it was working without that previously, I'm using https://github.com/DataBiosphere/leonardo/pull/1707 as a reference here.

Reply:
Working now, see https://codecov.io/gh/broadinstitute/rawls/commit/5859a3627b50c6e210201743daaf1793ef7da80d/

Reply:
Added status reporting (not via comment!) so we have some idea that codecov is happening and it's not totally invisible